app/assets/stylesheets/schedule.scss.erb

```diff
@@ -72,6 +106,37 @@
 .garden-day-header {
   padding-left: 20px;
   padding-top: 10px;
+  margin-bottom: 0;
   text-transform: uppercase;
   font-size: smaller;
 }
+
+.garden-selector {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  background-color: #C2D0CB;
+  border-radius: 0;
+  h3 {
+    margin-bottom: 10px;
+    margin-right: 20px;
+  }
+}
+
+.select-garden {
+  margin: 0 0.5rem 0 0.5rem;
+  padding: 12px;
+  background-color: rgba(240, 154, 60, 0.8);
+  border-radius: 10px;
+  font-weight: bold;
+  text-transform: uppercase;
+    &:hover {
+      background-color: rgba(209, 193, 189, 0.8);
```

Comment:
The hovers are a bit confusing, as we discussed in person. Consider, maybe, a drop shadow or a slight border or tint?

Reply:
Agreed! Updated color and added a box shadow on hover so it's more clear.